json-schema/json-schema-circe/src/main/scala/endpoints/circe/JsonSchemas.scala

```diff
@@ -100,6 +102,9 @@ trait JsonSchemas
     }
   }
 
+  def lazyRecord[A](schema: => Record[A], name: String): JsonSchema[A] = lazySchema(schema, name)
```

Comment:
Should they return a record / tagged ?

Reply:
I believe returning a `JsonSchema` is enough. We can change that later, if needed.